dags/glam.py

```diff
@@ -353,3 +394,9 @@
 client_scalar_probe_counts >> extracts_per_channel
 scalar_percentiles >> extracts_per_channel
 histogram_percentiles >> extracts_per_channel
+
+extracts_per_channel >> glam_import_desktop_aggs_beta
+extracts_per_channel >> glam_import_desktop_aggs_nightly
+extracts_per_channel >> glam_import_desktop_aggs_release
+extracts_per_channel >> glam_import_user_counts
+extracts_per_channel >> glam_import_probes
```

Comment:
@robhudson , Just wanted to check, is it fine if all the import jobs run parallel?
Currently it looks like it runs sequentially.

Reply:
Good catch. Parallel would be better actually. Some can take quite a while and I believe they were parallel previously via the crons.